tests/api/Jalasoft.TeamUp.Projects.API.Tests/PatchProjectByIdTest.cs

```diff
@@ -0,0 +1,76 @@
+﻿namespace Jalasoft.TeamUp.Projects.API.Tests
+{
+    using System;
+    using System.IO;
+    using Jalasoft.TeamUp.Projects.API.Controllers;
+    using Jalasoft.TeamUp.Projects.Core.Interfaces;
+    using Jalasoft.TeamUp.Projects.Models;
+    using Microsoft.AspNetCore.Http;
+    using Microsoft.AspNetCore.Mvc;
+    using Moq;
+    using Xunit;
+
+    public class PatchProjectByIdTest
+    {
+        private readonly Mock<IProjectsService> mockProjectsService;
+        private readonly DefaultHttpContext mockHttpContext;
+        private readonly PatchProjectById updateProject;
+
+        public PatchProjectByIdTest()
+        {
+            this.mockProjectsService = new Mock<IProjectsService>();
+            this.mockHttpContext = new DefaultHttpContext();
+            this.updateProject = new PatchProjectById(this.mockProjectsService.Object);
+        }
+
+        [Fact]
```

Comment:
please use a theory to send different use successful use cases, to the same for other scenarios